prince-of-versions/src/main/java/co/infinum/princeofversions/common/VersionContext.java

```diff
@@ -49,22 +50,28 @@
     private Map<String, String> metadata;
 
     /**
+     * Former minSdk value
+     */
+    private int minVersionMinSdk;
```

Comment:
I suggested that change and you even thumbed up :)

Problem here is only javadoc I guess as it not describe what this field actually is. This field is `minimum_version_min_sdk` from json config file. We just need better javadoc here I think.